src/Api.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+async function UserFolderData() {
+  const response = await fetch("https://bootcamp-api.codeit.kr/api/sample/folder" );
+  const data = await response.json();
+  return data;
+}
+
+ async function UserIdData() {
+  const response = await fetch("https://bootcamp-api.codeit.kr/api/sample/user");
+  const data = await response.json();
+  return data;
+}
+
+export{UserFolderData, UserIdData}
```

Comment:
[중요도: 중간] 
```suggestion
export async function UserFolderData() {
  const response = await fetch("https://bootcamp-api.codeit.kr/api/sample/folder" );
  const data = await response.json();
  return data;
}

export async function UserIdData() {
  const response = await fetch("https://bootcamp-api.codeit.kr/api/sample/user");
  const data = await response.json();
  return data;
}

```
굳이 마지막에 export를 적을 필요는 없습니다. 이 안의 코드들이 많아지면 많아질 수록 함수 하나 만들고 밑에 적기 귀찮기도 하구요... 함수에 직접 export하는 방법도 괜찮으니 추천드립니다.

Reply:
공부하면서 함수를 한번에 모아서 export 하는 방법이 깔끔하다 생각했는데 멘토님 말씀대로 코드가 많아지면 번거로울 수 있다고 생각을 못했습니다.
수정했습니다!
https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/185/commits/d02f98406d5fd3078bd1d85afa5a74a77c1c332d